models/wholeBody_ct_segmentation/docs/README.md

```diff
@@ -0,0 +1,166 @@
+
+## 1.5 mm (highres) model (Single Model with 104 foreground classes)
+
+Benchmarking on GPU: Memory: **28.73G**
+
+- `++ Latencies => Total: 6.0277; Pre: 1.6228; Inferer: 4.1153; Invert: 0.0000; Post: 0.0897; Write: 0.1995`
+
+Benchmarking on CPU: Memory: **26G**
+
+- `++ Latencies => Total: 38.3108; Pre: 1.6643; Inferer: 30.3018; Invert: 0.0000; Post: 6.1656; Write: 0.1786`
+
+## 3.0 mm (lowres) model (single model with 104 foreground classes)
+
+GPU: Memory: **5.89G**
+
+ - `++ Latencies => Total: 1.9993; Pre: 1.2363; Inferer: 0.5207; Invert: 0.0000; Post: 0.0358; Write: 0.2060`
+
+CPU: Memory: **2.3G**
+
+ - `++ Latencies => Total: 6.6138; Pre: 1.3192; Inferer: 3.6746; Invert: 0.0000; Post: 1.4431; Write: 0.1760`
+
+## Performance
+
+- 1.5 mm Model Training
+
+  - Training Accuracy
+
+![](./imgs/totalsegmentator_train_accuracy.png) <br>
+
+  - Validation Dice
+
+![](./imgs/totalsegmentator_15mm_validation.png) <br>
+
+## MONAI Bundle Commands
+In addition to the Pythonic APIs, a few command line interfaces (CLI) are provided to interact with the bundle. The CLI supports flexible use cases, such as overriding configs at runtime and predefining arguments in a file.
+
+For more details usage instructions, visit the [MONAI Bundle Configuration Page](https://docs.monai.io/en/latest/config_syntax.html).
+
+#### Execute training
+
+```
+python -m monai.bundle run training --meta_file configs/metadata.json --config_file configs/train.json --logging_file configs/logging.conf
+```
+
+#### Override the `train` config to execute multi-GPU training
+
+```
+torchrun --standalone --nnodes=1 --nproc_per_node=2 -m monai.bundle run training --meta_file configs/metadata.json --config_file "['configs/train.json','configs/multi_gpu_train.json']" --logging_file configs/logging.conf
+```
+
+Please note that the distributed training-related options depend on the actual running environment; thus, users may need to remove `--standalone`, modify `--nnodes`, or do some other necessary changes according to the machine used. For more details, please refer to [pytorch's official tutorial](https://pytorch.org/tutorials/intermediate/ddp_tutorial.html).
+
+#### Override the `train` config to execute evaluation with the trained model
+
+```
+python -m monai.bundle run evaluating --meta_file configs/metadata.json --config_file "['configs/train.json','configs/evaluate.json']" --logging_file configs/logging.conf
+```
+
+#### Override the `train` config and `evaluate` config to execute multi-GPU evaluation
+
+```
+torchrun --standalone --nnodes=1 --nproc_per_node=2 -m monai.bundle run evaluating --meta_file configs/metadata.json --config_file "['configs/train.json','configs/evaluate.json','configs/multi_gpu_evaluate.json']" --logging_file configs/logging.conf
+```
+
+#### Execute inference
+
+```
+python -m monai.bundle run evaluating --meta_file configs/metadata.json --config_file configs/inference.json --logging_file configs/logging.conf
```

Comment:
Please consider adding the support/documentation of running model inference with CLI on a single file or a folder of image files. Cc @nvahmadi 

Something like:

python -m monai.bundle run evaluating \
    --config_file configs/inference.json \ 
    --datalist "['/test/IBSR_01_ana.nii.gz', '/Users/Documents/MNI152_T1_2mm.nii.gz']"

Reply:
Hi, thanks for Cc'ing me, @wyli . I recently discussed multi-image/folder inference using a trained MONAI bundle, but that was in the context of python-based inference, i.e. not CLI. For the CLI, I think the solution is to pass a datalist parameter (as @wyli indicated above), or a dataset.json file. For python-based inference, I have a code snippet that may help. If you're interested @tangy5, please let me know.

Reply:
Hi @wyli @nvahmadi ,

So I think the PR is already good enough?
You can override the `datalist` of the `inference.json` config at runtime CLI:
```json
"datalist": "$list(sorted(glob.glob(@dataset_dir + '/imagesTs/*.nii.gz')))"
```

Thanks.

Reply:
Hi @Nic-Ma, thanks, I think this example is good for indicating how to query a folder for `.nii.gz` files. But especially for new users, the documentation would benefit from a concrete example with filepaths, similar to what @wyli indicated (`--datalist "['/test/IBSR_01_ana.nii.gz', '/Users/Documents/MNI152_T1_2mm.nii.gz']"`). Would be good to include that. Other than that, I have no comments w.r.t. the PR.

Reply:
Thanks. I think the idea is good. 
To simplify the process for running the bundle inference. 
Also similar to MONAI Label usage, we need more instructions. 

Please suggest any code snippet to support this. If only doc is needed to change, we can update the PR/bundle.

Reply:
I think one of the key features of totalsegmentator is that the command interface is very easy to use... Shall we polish our CLI inference usage as well? For example, why a user would care about the --meta_file option, how to run it in cpu mode or in --fast mode, how to only output a subset of predictions?

Reply:
sure, that'll be great if you want to improve the CLI usability for all the bundles. For this particular bundle,  when compared with https://github.com/wasserth/TotalSegmentator#usage  do you think we should also review the CLI usage as well?

Reply:
Hi @wyli @Nic-Ma , thanks for the discussions. I will spend some time to check TotalSegmentator first. As for the potential enhancements of CLI usability, if it's need I will create a separate ticket.

Reply:
Thanks, please help create a separate ticket, so that we are aware of the usability gaps.